src/lando/main/management/commands/__init__.py

```diff
@@ -1,6 +1,6 @@
 from time import sleep
 
-from lando.main.models import Worker
+from lando.main.models.base import Worker
```

Comment:
This will need to be changed based on the other PR.

Reply:
Ah, let me merge those other ones first and then I will request re-review after updating this one.